server/svix-server/src/queue/redis.rs

```diff
@@ -34,16 +36,15 @@ use super::{
 
 // FIXME: Change unwraps to have our own error type for the queue module entirely
 
-const MAIN: &str = "svix_queue_main";
-const PROCESSING: &str = "svix_queue_processing";
-const DELAYED: &str = "svix_queue_delayed";
+// FIXME: hash-tags here enable clustering but prevent proper sharding of lists
+const MAIN: &str = "{svix_queue}_main";
+const PROCESSING: &str = "{svix_queue}_processing";
+const DELAYED: &str = "{svix_queue}_delayed";
```

Comment:
This change is quite problematic, because you may have items in the queue that you will now won't be pulling due to this (in the non-clustered case).

I'm talking about someone using Svix for a few weeks now, and now upgrading.

Reply:
Migrations should address this.